File: python/muse/algorithms/meta/Sorting.py
from muse.algorithms.sorting import BubbleSort
from muse.algorithms.sorting import InsertionSort
from muse.algorithms.sorting import MergeSort
from muse.algorithms.sorting import QuickSort
from muse.algorithms.sorting import SelectionSort

def bubbleSort(arr: list) -> None:
    BubbleSort.sort(arr)

def insertionSort(arr: list) -> None:
    InsertionSort.sort(arr)

def mergeSort(arr: list) -> None:
    MergeSort.sort(arr)

def quickSort(arr: list) -> None:
    QuickSort.sort(arr)

def selectionSort(arr: list) -> None:
    SelectionSort.sort(arr)

File: nodejs/src/muse/algorithms/meta/Sorting.js
"use strict"

const BubbleSort    = require("muse/algorithms/sorting/BubbleSort")
const InsertionSort = require("muse/algorithms/sorting/InsertionSort")
const MergeSort     = require("muse/algorithms/sorting/MergeSort")
const QuickSort     = require("muse/algorithms/sorting/QuickSort")
const SelectionSort = require("muse/algorithms/sorting/SelectionSort")

exports.bubbleSort = (arr) => {
    BubbleSort.sort(arr)
}

exports.insertionSort = (arr) => {
    InsertionSort.sort(arr)
}

exports.mergeSort = (arr) => {
    MergeSort.sort(arr)
}

exports.quickSort = (arr) => {
    QuickSort.sort(arr)
}

exports.selectionSort = (arr) => {
    SelectionSort.sort(arr)
}
